### PR TITLE
add supports/plugin data for gradient interpolation and color-mix

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changes to cssdb
 
+### Unreleased (patch)
+
+- Updated support date for `color-mix`
```
Comment (Antonio-Laguna) [inline CHANGELOG.md:5]: Is this date or data?

Comment (romainmenke) [reply CHANGELOG.md:5]: Good catch!

Comment (romainmenke) [inline CHANGELOG.md:5]:
```suggestion
- Updated support data for `color-mix`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changes to cssdb
 
+### Unreleased (patch)
+
+- Updated support data for `color-mix`
+- Updated support date for `gradients-interpolation-method`
```
Comment (romainmenke) [inline CHANGELOG.md:6]:
```suggestion
- Updated support data for `gradients-interpolation-method`
```

**Review by Antonio-Laguna — APPROVED** [review]
LGTM!

**Pull request merged**